[2024-10-12T19:27:38.213+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-12T19:27:38.225+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow.get_age manual__2024-10-12T19:27:37.188960+00:00 [queued]>
[2024-10-12T19:27:38.230+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow.get_age manual__2024-10-12T19:27:37.188960+00:00 [queued]>
[2024-10-12T19:27:38.230+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-12T19:27:38.238+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): get_age> on 2024-10-12 19:27:37.188960+00:00
[2024-10-12T19:27:38.243+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow', 'get_age', 'manual__2024-10-12T19:27:37.188960+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/taskflow.py', '--cfg-path', '/tmp/tmp03v_rpoh']
[2024-10-12T19:27:38.245+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1620) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-12T19:27:38.245+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask get_age
[2024-10-12T19:27:38.245+0000] {standard_task_runner.py:72} INFO - Started process 1625 to run task
[2024-10-12T19:27:38.273+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow.get_age manual__2024-10-12T19:27:37.188960+00:00 [running]> on host 91a8a8c9b949
[2024-10-12T19:27:38.322+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='james' AIRFLOW_CTX_DAG_ID='dag_with_taskflow' AIRFLOW_CTX_TASK_ID='get_age' AIRFLOW_CTX_EXECUTION_DATE='2024-10-12T19:27:37.188960+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-12T19:27:37.188960+00:00'
[2024-10-12T19:27:38.323+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-12T19:27:38.328+0000] {python.py:240} INFO - Done. Returned value was: 29
[2024-10-12T19:27:38.343+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-12T19:27:38.344+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow, task_id=get_age, run_id=manual__2024-10-12T19:27:37.188960+00:00, execution_date=20241012T192737, start_date=20241012T192738, end_date=20241012T192738
[2024-10-12T19:27:38.384+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-12T19:27:38.396+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-12T19:27:38.397+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
